b2/console_tool.py

```diff
@@ -1008,9 +1016,13 @@ def run_command(self, argv):
             logger.info('ConsoleTool \'args is None\' - printing usage')
             self._print_stderr(command.command_usage())
             return 1
-        elif args.logConfig and args.debugLogs:
-            logger.info('ConsoleTool \'args.logConfig and args.debugLogs\' were both specified')
-            self._print_stderr('ERROR: --logConfig and --debugLogs cannot be used at the same time')
+        elif [args.logConfig, args.verbose, args.debugLogs].count(True) > 1:
```

Comment:
You should use a tuple here, which is faster to initialize and doesn't need to allocate memory ahead to amortize `,append()` calls.

`.count()` usage here is very clever!

Reply:
Pawel is correct that tuples are faster, but this only happens once per invocation of the command-line tool, so I'm not worried about it.  I'm going to go ahead and merge the pull request.

Reply:
@ppolewicz Thanks, python isn't my main language so advice like this is helpful. I'll use a tuple next time!